patches/master_patch.patch

```diff
@@ -201,6 +281,105 @@ index 247d24220beb7d176b890d7c2d45bdd2dfd8db99..c270bba0fabb557e8d3d1df8ac166bbb
  
    ProfileWriter* const writer_;  // weak
    const base::WeakPtr<ExternalProcessImporterHost> host_;
+diff --git a/chrome/browser/interstitials/chrome_metrics_helper.cc b/chrome/browser/interstitials/chrome_metrics_helper.cc
+index 860c919d1a656acf63a5a0875484d59d5dd40515..35c9a27119148fbce09e70771d301a3817a932f2 100644
+--- a/chrome/browser/interstitials/chrome_metrics_helper.cc
++++ b/chrome/browser/interstitials/chrome_metrics_helper.cc
+@@ -12,10 +12,6 @@
+ #include "content/public/browser/web_contents.h"
+ #include "extensions/features/features.h"
+ 
+-#if BUILDFLAG(ENABLE_EXTENSIONS)
+-#include "chrome/browser/extensions/api/experience_sampling_private/experience_sampling.h"
+-#endif
+-
+ #if BUILDFLAG(ENABLE_CAPTIVE_PORTAL_DETECTION)
+ #include "chrome/browser/ssl/captive_portal_metrics_recorder.h"
+ #endif
+@@ -59,58 +55,8 @@ void ChromeMetricsHelper::RecordExtraShutdownMetrics() {
+ 
+ void ChromeMetricsHelper::RecordExtraUserDecisionMetrics(
+     security_interstitials::MetricsHelper::Decision decision) {
+-#if BUILDFLAG(ENABLE_EXTENSIONS)
```

Comment:
we can use `#if BUILDFLAG(ENABLE_EXTENSIONS) && defined(MUON_CHROMIUM_BUILD)` to reduce the patch size